libcontainer/rootfs_linux.go

```diff
@@ -339,13 +339,17 @@ func mountToRootfs(m *configs.Mount, rootfs, mountLabel string, enableCgroupns b
 		return label.SetFileLabel(dest, mountLabel)
 	case "tmpfs":
 		copyUp := m.Extensions&configs.EXT_COPYUP == configs.EXT_COPYUP
+		isRo := m.Flags&unix.MS_RDONLY == 1
 		tmpDir := ""
 		stat, err := os.Stat(dest)
 		if err != nil {
 			if err := os.MkdirAll(dest, 0755); err != nil {
 				return err
 			}
 		}
+		if isRo {
+			m.Flags &= ^unix.MS_RDONLY
```

Comment:
m shouldn't be altered, please copy the struct

Reply:
I have changed a little bit the code. Now I am reusing a remount function that was in the code, so I did not require to copy the struct. Thanks @AkihiroSuda